### PR TITLE
Run a11y and lighthouse tests through a11y-tests-web

### PR DESCRIPTION
**Pull request by radiocontrolled**
Resolves https://github.com/bbc/simorgh/issues/331

This PR: 
* uses a11y-test-web's version of bbc-a11y
** as such, removes bbc-a11y as core dependency for Simorgh 
* adds lighthouse testing for accessibility, pwa, seo and best-practices
* ensure the above run on travis

- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +33,10 @@
     "test:e2e:storybook:ci": "run-p --race storybook cypress:storybook",
     "test:e2e:storybook:interactive": "npm run killStorybook && run-p --race storybook cypress:storybook:interactive",
     "test:lighthouse": "npm run killApp && npm run build && run-p --race start lighthouse",
+    "test:a11y-tests-web:bbc-a11y" : "wait-on http://localhost:7080/ && A11Y_CONFIG=simorgh/simorgh A11Y_SIMORGH_BASE_URL=http://localhost:7080 npm --prefix ./node_modules/@bbc/a11y-tests-web run start:bbc-a11y:headless",    
+    "test:a11y-tests-web:bbc-a11y:ci" : "npm run build && run-p --race start test:a11y-tests-web:bbc-a11y",
+    "test:a11y-tests-web:lighthouse": "wait-on http://localhost:7080/ &&  A11Y_CONFIG=simorgh/simorgh OUTPUT_JSON='/../../../../reports/lighthouse-report.json' npm --prefix ./node_modules/@bbc/a11y-tests-web run start:lighthouse:junit",
```
Comment (radiocontrolled) [inline package.json]: This `/../../../../` is hard to read. Want to use `pwd` instead, but unclear how to pass this as a parameter.

**Comment by radiocontrolled** [comment]
This PR is blocked because it is waiting on a review of https://github.com/bbc/a11y-tests-web/pull/19

**Comment by jamesbhobbs** [comment]
@jtart please update this in Alison's absence

**Comment by radiocontrolled** [comment]
Blocked as this is waiting for a code review.

**Comment by radiocontrolled** [comment]
Working on a related PR
See https://github.com/BBC-News/simorgh/compare/node-lighouse-travis

**Pull request closed without merging**